jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/EtcdContainer.java

```diff
@@ -108,6 +109,10 @@ protected void configure() {
             return;
         }
 
+        if (System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("linux")) {
+            withCreateContainerCmdModifier(cmd -> cmd.withUser("1000"));
```

Comment:
Can we provide an option to control whether to run in rootless mode? with the true by default.

And userID also have an option would be great.

Reply:
- I uploaded another commit. But to be honest, `user` is actually `uid:gid`, not just `uid`. And I'm not sure if there is a concept of uid:gid on windows/mac.

> [ user | user:group | uid | uid:gid | user:gid | uid:group ]

- This is the low-level Java API for https://github.com/docker-java/docker-java . So I would say I don't know if there is a similar equivalent for podman either.